_data/master/navbars/reference.yml

```diff
@@ -113,8 +113,19 @@ toc:
     path: /reference/architecture/data-path
 - title: Advanced
   section:
-  - title: RBAC for etcdv2
-    path: /reference/advanced/etcd-rbac
+  - title: Guide for etcd RBAC
```

Comment:
Not sure about this title

Perhaps something like "Configuring RBAC using etcdv2"